packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.ts

```diff
@@ -12769,7 +12837,16 @@ module ZIndexes {
 // WARNING: Unsupported export: FocusZoneTabbableElements
 // WARNING: Unsupported export: FocusZoneTabbableElements
 // WARNING: Unsupported export: Grid
+// WARNING: Unsupported export: IGroupHeaderStyleProps
+// WARNING: Unsupported export: IGroupFooterStyleProps
+// WARNING: Unsupported export: IGroupShowAllStyleProps
 // WARNING: Unsupported export: GroupedList
+// WARNING: Unsupported export: GroupHeader
+// WARNING: Unsupported export: GroupFooter
+// WARNING: Unsupported export: GroupShowAll
+// WARNING: Unsupported export: SPACER_WIDTH

```

Comment:
We may want to use named exports for `GroupSpacer` otherwise we'll be exporting `SPACER_WIDTH` as a top-level export (part of our public API).